indigo_tasks/models.py

```diff
@@ -4,48 +4,63 @@
 from django.db import models
 
 from indigo_api.models import Country, Locality, Work, Document
+from django.contrib.auth.models import User
+
+import datetime
 
 
 class Task(models.Model):
-    title = models.CharField(max_length=256)
-    content = models.TextField()
+    title = models.CharField(max_length=256, null=False, blank=False, default='New task: %s' % (datetime.date.today()))
+    content = models.TextField(null=True, blank=True)
 
     country = models.ForeignKey(Country, related_name='tasks', null=False, blank=False, on_delete=models.CASCADE)
     locality = models.ForeignKey(Locality, related_name='tasks', null=True, blank=True, on_delete=models.CASCADE)
     work = models.ForeignKey(Work, related_name='tasks', null=True, blank=True, on_delete=models.CASCADE)
-    document = models.ForeignKey(Document, related_name='tasks', null=True, blank=True)
-    anchor = models.CharField(max_length=128, null=True, blank=True)
+    document = models.ForeignKey(Document, related_name='tasks', null=True, blank=True, on_delete=models.CASCADE)
+
+    # cf indigo_api.models.Annotation
+    anchor_id = models.CharField(max_length=128, null=True, blank=True)
+
+    state = models.CharField(max_length=128, null=True, blank=True, choices=(
```

Comment:
I've made `null` and `blank = False` here rather

Reply:
Good call.